src/parseGithubWebhook.js

```diff
@@ -0,0 +1,27 @@
+class ParseGithub {
```

Comment:
Let's convert this to a regular function, Seams out of place to have a single Class with no internal functions.